recipes/patchelf/all/test_package/conanfile.py

```diff
@@ -0,0 +1,9 @@
+from conans import ConanFile, CMake, tools
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+
+    def test(self):
+        if not tools.cross_building(self.settings, skip_x64_x86=True):
```

Comment:
```suggestion
        if not tools.cross_building(self, skip_x64_x86=True):
```

Reply:
https://github.com/conan-io/hooks/issues/324 spreading the word!